src/main/resources/assets/admin/common/js/ui/time/Picker.ts

```diff
@@ -187,13 +202,25 @@ export class Picker<T extends Element>
         }
     }
 
+    private initDefaultButton() {
+        this.popupDefaultButton = new Button(i18n('action.setDefault'));
+        this.popupDefaultButton.addClass('default-button');
+        this.popupDefaultButton.onClicked(() => {
+            if (this.setDefaultHandler) {
+                this.defaultHandler();
+            }
```

Comment:
Condition `if (this.setDefaultHandler) `doesn't make sense. This method is defined in the class, so it always exists.

````
private initDefaultValueButton() {
  ...
  const defaultValueButton = ...;
  defaultValueButton.onClicked(() => this.defaultValueHandler());
  return defaultValueButton;
}